ktor-http/common/test/io/ktor/tests/http/DataUrlTest.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2014-2024 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.tests.http
+
+import io.ktor.http.*
+import kotlin.test.*
+
+class DataUrlTest {
```

Comment:
Probably we should add tests into `UrlTest` instead. And add at least one forth-and-back test verifying URL rendering works as expected.

Reply:
thanks fixed